opts/opts.go

```diff
@@ -16,7 +16,7 @@ var (
 	alphaRegexp  = regexp.MustCompile(`[a-zA-Z]`)
 	domainRegexp = regexp.MustCompile(`^(:?(:?[a-zA-Z0-9]|(:?[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9]))(:?\.(:?[a-zA-Z0-9]|(:?[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])))*)\.?\s*$`)
 	// DefaultHTTPHost Default HTTP Host used if only port is provided to -H flag e.g. docker daemon -H tcp://:8080
-	DefaultHTTPHost = "127.0.0.1"
+	DefaultHTTPHost = "localhost"
```

Comment:
The reason why I change `DefaultHTTPHost` from `127.0.0.1` to `localhost` is that the ca in `integration-cli/fixtures/https` only validate certificate for `localhost`, but not for `127.0.0.1`, if we update ca in `integration-cli/fixtures/https` validate certificate both for `localhost` and `127.0.0.1`, this change is not necessary.